test/test_playbooks/composite_content_view.yml

```diff
@@ -0,0 +1,17 @@
+---
+- hosts: localhost
+  tasks:
+  - name: Load server config
+    include_vars:
+      file: server_vars.yml
+  - name: "Create/update katello composite content view"
+    katello_content_view:
+      username: "{{ foreman_username }}"
+      password: "{{ foreman_password }}"
+      server_url: "{{ foreman_server_url }}"
+      verify_ssl: "{{ foreman_verify_ssl }}"
+      name: "ccv5"
+      organization: "Default Organization"
+      content_views:
+        - name: "cv3"
+          version "1.0"
```

Comment:
You can add the test to the test suite (of PR #62 ) already.
https://github.com/theforeman/foreman-ansible-modules/blob/412507090f314263b774aa2e82ca47fa4ff9cfee/test/test_crud.py#L5
You can then run the test with `make test TEST="-k composit_content_view"`.

Reply:
ok. but what about the dependencies. it needs the products, repos, cv, and cv version to exist

Reply:
You can look at the Product playbook. It creates the needed Organization first and removes it again afterwards.

Reply:
did that now... heh.. :D